### PR TITLE
Add dependabot config

### PR DESCRIPTION
**Pull request by kaloyanm**

### CODE REVIEW

**Comment by i-trofimtschuk** [comment]
tests fail ^^

**Comment by i-trofimtschuk** [comment]
> tests fail ^^

travis config PR #10 should fix tests

**Comment by i-trofimtschuk** [comment]
closed + reopened to retriggfer travis

File: .dependabot/config.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+update_configs:
+  # python updates
+  - package_manager: "python"
+    directory: "/"
+    version_requirement_updates: "auto"
+    default_reviewers:
+      - "i-trofimtschuk"
+      - "vasilty"
+      - "kaloyanm"
+      - "djangsters/backend"
```
Comment (i-trofimtschuk) [inline .dependabot/config.yml]: @Chronial do you want to also be on the list of reviewers by default?

File: .dependabot/config.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    update_schedule: "weekly"
+    allowed_updates:
+      - match:
+          update_type: "security"
+      - match:
+          dependency_type: "direct"
```
Comment (vasilty) [inline .dependabot/config.yml]: new line missing

Comment (i-trofimtschuk) [reply .dependabot/config.yml]:
```suggestion
          dependency_type: "direct"

```

**Pull request merged**